static/sb-alg-list.json

```diff
@@ -0,0 +1,158 @@
+[
```

Comment:
DELETE ME

Reply:
This file should not have been committed to the branch.  That was a mistake.